resources/views/checkout/partials/address.blade.php

```diff
@@ -145,14 +145,30 @@
                 />
             </label>
         </div>
-        <div class="col-span-12 sm:col-span-6">
+        <div class="col-span-12 sm:col-span-6 sm:col-start-1">
             <label>
                 <x-rapidez::label>@lang('Country')</x-rapidez::label>
                 <x-rapidez::input.select.country
                     name="{{ $type }}_country"
                     dusk="{{ $type }}_country"
                     v-model="variables.country_code"
-                    v-on:change="$root.$nextTick(() => window.app.$emit('postcode-change', variables))"
+                    v-on:change="$root.$nextTick(() => {
+                        window.app.$emit('postcode-change', variables);
+                        variables.region_id = null
+                    })"
+                    required
+                />
+            </label>
+        </div>
+        <div class="col-span-12 sm:col-span-6 has-[.exists]:block hidden">
+            <label>
+                <x-rapidez::label>@lang('Region')</x-rapidez::label>
+                <x-rapidez::input.select.region
+                    class="exists"
+                    name="{{ $type }}_region"
+                    dusk="{{ $type }}_region"
+                    country="variables.country_code"
+                    v-model="variables.region_id"
                     required
```

Comment:
Perhaps we'll want to listen to 
![image](https://github.com/user-attachments/assets/b7a1b7c4-dda7-4c91-9a5e-0cbf3ae5aee3)
for the required attribute
`general/region/state_required`
and
`general/region/display_all`